avm/res/cache/redis/linked-servers/main.bicep

```
@@ -0,0 +1,44 @@
+metadata name = 'Redis Cache Linked Servers'
+metadata description = 'This module connects a primary and secondary Redis Cache together for geo-replication.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. Primary Redis cache name.')
+param redisCacheName string
+
+@description('Required. The name of the secondary Redis cache. If not provided, the primary Redis cache name is used.')
```

```suggestion
@description('Optipnal. The name of the secondary Redis cache. If not provided, the primary Redis cache name is used.')
```

With the default it's now actually optional :)

Please note, after applying all changes I'd recommend to run the `Set-AVMModule -Recurse` script one more time

---

Fixed.